articles/sts_storage_locations.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "STS Storage Locations"
```

Comment:
```suggestion
title: "AWS Security Token Service Storage Locations"
```

Reply:
I would rename the file to `aws_sts_storage_locations.md` as well for linking purposes.